src/FlexCell/FlexCell.levers.js

```diff
@@ -0,0 +1,29 @@
+/**
+ * Available HTML elements of the flex cell.
+ */
+export const flexCellElements = ['div', 'section', 'article', 'main', 'footer', 'aside', 'header', 'nav'];
+
+/**
+ * Width of the flex cell (n-of-12 columns).
+ */
+export const flexCellWidth = [1, 2, 3, 4, 5, 6, 7, 8, 9, 10, 11, 'full', 'auto'];
```

Comment:
Why not include 12 here?

Reply:
Oh, it's a TW thing. They have w-n/12 classes from w-1/12 up to w-11/12, but instead of w-12/12, they have w-full instead.
https://tailwindcss.com/docs/width